build/pause/Dockerfile.Rhel

```diff
@@ -0,0 +1,12 @@
+FROM registry.ci.openshift.org/ocp/builder:ubi8.art AS builder
+WORKDIR /go/src/github.com/openshift/kubernetes/build/pause
+COPY . .
+RUN dnf install -y gcc glibc-static && \
+    gcc -Os -Wall -Werror -static -o bin/pause ./linux/pause.c
+
+FROM scratch
+COPY --from=builder /go/src/github.com/openshift/kubernetes/build/pause/bin/pause /usr/bin/
+LABEL io.k8s.display-name="OpenShift Pod" \
+      io.k8s.description="This is a component of OpenShift and contains the binary that holds the pod namespaces." \
+      io.openshift.tags="openshift"
+ENTRYPOINT [ "/usr/bin/pause" ]
```

Comment:
fwiw  this is currently incompatible with MCO https://github.com/openshift/machine-config-operator/blob/master/templates/master/01-master-container-runtime/_base/files/crio.yaml#L30
should we also symlink to /usr/bin/pod to allow a graceful transition?

Reply:
Yes, good catch! Will add a symlink

Reply:
Can't create a symlink because we are building from `scratch` and we don't preserve symlink relation from one build stage to another. So simple solution, I just copied the pause binary to `/usr/bin/pod` so we don't break anything.